```
generated/linux/release/64/benchmark.o:benchmark/runbench.d:_D32TypeInfo_S3std4file11DirIterator6__initZ: error: undefined reference to '_D3std4file11DirIterator9__xtoHashFNbNeKxSQBoQBnQBlZm'
generated/linux/release/64/benchmark.o:benchmark/runbench.d:_D32TypeInfo_S3std4file11DirIterator6__initZ: error: undefined reference to '_D3std4file11DirIterator11__xopEqualsFKxSQBnQBmQBkKxQmZb'
collect2: error: ld returned 1 exit status
```

No idea why. Try force pushing or rebasing?

--- a/src/dmd/statementsem.d
+++ b/src/dmd/statementsem.d
@@ -2691,7 +2691,7 @@ private extern (C++) final class StatementSemanticVisitor : Visitor
                 ed = ds.isEnumDeclaration(); // typedef'ed enum
             if (!ed && te && ((ds = te.toDsymbol(sc)) !is null))
                 ed = ds.isEnumDeclaration();
-            if (ed)
+            if (ed && ss.cases.length < ed.members.length)
             {
                 int missingMembers = 0;
                 const maxShown = !global.params.verbose ? 6 : int.max;

